src/Microsoft.XmlSerializer.Generator/tests/SGenTests.cs

```diff
@@ -5,16 +5,155 @@
 
 using Xunit;
 using Microsoft.XmlSerializer.Generator;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System;
 
 namespace Microsoft.XmlSerializer.Generator.Tests
 {
     public static class SgenTests
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
```

Comment:
Instead of adding `SkipOnTargetFramework` on each method, we can exclude the entire file when targeting `uap`.